mtags/src/main/scala/scala/meta/internal/mtags/CommonMtagsEnrichments.scala

```diff
@@ -518,12 +518,15 @@ trait CommonMtagsEnrichments {
      */
     def isScalaOrJava: Boolean = {
       toLanguage match {
-        case Language.SCALA | Language.JAVA =>
-          !Files.exists(path.toNIO) ||
-          Files.isRegularFile(path.toNIO)
+        case Language.SCALA | Language.JAVA => isFile
```

Comment:
> I'm certain that I saw this kind of fix recently. Is is another place?

The fix was here and I didn't notice we had a single `isScala` and `isJava` also